.env.example

```diff
@@ -0,0 +1,21 @@
+# Dotenv
+#
+# This file commits safe environment variables for the development environment.
+# For managing sensitive values and overrides use `/.env.development.local`
+#
+# Reference: https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
+
+# Rollbar for application monitoring
+ROLLBAR_ACCESS_TOKEN=ROLLBAR_ACCESS_TOKEN
+ROLLBAR_ENV=development
+
+# TODO: Replace `rails-template` with the name of the app.
+DATABASE_URL=postgres://postgres@localhost:5432/rails-template-development
```

Comment:
And is it worth TODOING this TODO like in `.env.test`?

Reply:
👍 yes, have done now